packages/forms/src/UIForm/fields/TimezoneList/TimezoneList.utils.js

```diff
@@ -0,0 +1,151 @@
+import get from 'lodash/get';
+
+import frTimezones from 'cldr-dates-full/main/fr/timeZoneNames.json';
+import enTimezones from 'cldr-dates-full/main/en/timeZoneNames.json';
+import jaTimezones from 'cldr-dates-full/main/ja/timeZoneNames.json';
+import deTimezones from 'cldr-dates-full/main/de/timeZoneNames.json';
+
+/**
+ * Get the offset between a timezone and the UTC time (in minutes)
+ * @param {String} timeZone Timezone IANA name
+ * @returns {Number}
+ * 
+ * @see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+ */
+export function getTimezoneUTCOffset(timeZone) {
+	// Build localized formats for UTC and the target timezone
+	const formatOptions = {
+		hourCycle: 'h23',
+		year: 'numeric',
+		month: 'numeric',
+		day: 'numeric',
+		hour: 'numeric',
+		minute: 'numeric'
+	};
+
+	const locale = 'en-US';
+	const utcFormat = new Intl.DateTimeFormat(locale, { ...formatOptions, timeZone: 'Etc/UTC' });
+	const timezoneFormat = new Intl.DateTimeFormat(locale, { ...formatOptions, timeZone });
+
+	// Create the same date in UTC timezone and the target timezone
+	const date = new Date();
+	const utcDate = new Date(utcFormat.format(date));
+	const timezoneDate = new Date(timezoneFormat.format(date));
+
+	// Compute delta between dates
+	return (timezoneDate - utcDate) / (1000 * 60);
+}
+
+/**
+ * Ensure a numeric value is expressed on two digits (01, 02, 10 ...)
+ * @param {Number} value
+ * @returns {String}
+ */
+function padTwoDigits(value) {
+	return value > 9 ? value : `0${value}`;
+}
+
+/**
+ * Format an UTC offset from minutes to [+/-][HH][separator][mm]
+ * @param {Number} offset UTC offset
+ * @param {String} separator Separator between hours and minutes
+ * @returns {String} Formatted UTC offset
+ */
+function formatUtcOffset(offset, separator) {
+	const sign = offset >= 0 ? '+' : '-';
+
+	const absoluteOffset = Math.abs(offset);
+	const min = absoluteOffset % 60;
+	const hours = (absoluteOffset - min) / 60;
+
+	return `${sign}${padTwoDigits(hours)}${separator}${padTwoDigits(min)}`;
+}
```

Comment:
Those helpers already exist in UI/components, there might be a better way to share it
- An independant date utils package seems overkill
- Direct import is dirty
- ?

Reply:
forms depends on components already

Reply:
@frassinier good point
@jmfrancois true, but utils are not exposed (and ... it's a components repo actually 😅 )

Reply:
If you create a package that is "shared" because we use UMD we need to make it a UMD to make it really shared.
Look at @talend/http it is embed every where so it is there multiple times.
So may be we should open a track on that on how to share code needed by components taking all constraints into account ?